### PR TITLE
AWS Rabbitmq configuration

### PR DESCRIPTION
**Pull request by afda16**
Add Rabbitmq configuration for AWS:
- Override hieradata settings to install latest package and use autocluster
- Install autocluster plugin

### CODE REVIEW

File: hieradata_aws/class/rabbitmq.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - rummager
+  - stagecraft
+
+govuk_safe_to_reboot::can_reboot: 'careful'
+govuk_safe_to_reboot::reason: 'rabbitmq-1 is a single point of failure for content-store, apps not resilient'
+
+rabbitmq::config_kernel_variables:
+  'inet_dist_listen_min': '9100'
+  'inet_dist_listen_max': '9105'
+rabbitmq::config_mirrored_queues: true
```
Comment (surminus) [inline hieradata_aws/class/rabbitmq.yaml]: Was this still required?

Comment (afda16) [inline hieradata_aws/class/rabbitmq.yaml]: Not sure ...

**Comment by afda16** [comment]
I removed the deprecated parameter

**Comment by surminus** [comment]
Feel free to merge.

**Pull request merged**